lib/workers/pr/changelog/release-notes.spec.ts

```diff
@@ -225,7 +225,7 @@ describe(getName(__filename), () => {
       'gets release notes with body from gitlab repo %s',
       async (prefix) => {
         httpMock
-          .scope('https://api.gitlab.com/')
+          .scope('https://gitlab.com/api/v4/')
```

Comment:
I changed this mock URL too, I came across it and it's misleading since there is no such thing as https://api.gitlab.com/